web/static/js/components/category_column.jsx

```diff
@@ -5,9 +5,9 @@ import styles from "./css_modules/category_column.css"
 
 function CategoryColumn(props) {
   const categoryToEmoticonUnicodeMap = {
```

Comment:
hey @tnewell5! thanks for picking this one up. Unfortunately, this implementation (for the moment) borks the action item column, as replacing the `i` tag outright with an img tag doesn't account for the action item icon. Rather than have you add a messy conditional, I'm wondering if @imaginista can cook up an SVG for the action item column as well. That way, we can move wholesale to image tags.